.gitlab/pipeline/check.yml

```diff
@@ -214,6 +214,7 @@ check-runtime-migration-collectives-westend:
     PACKAGE: "collectives-westend-runtime"
     WASM: "collectives_westend_runtime.compact.compressed.wasm"
     URI: "wss://westend-collectives-rpc.polkadot.io:443"
+    COMMAND_EXTRA_ARGS: "--disable-spec-name-check"
```

Comment:
This can be removed after the collectives was upgraded?

Reply:
Yes